cli/core/pkg/common/config_client_interface.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2021 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+package common
```

Comment:
`common` is a very generic package and eventually developers tend to put random stuff to this type of package because of no defined boundaries. 

For `config_client_interface.go` and `http_client_interface.go`, If we are expecting more interfaces like this in future, should we create a separate package `interface` instead of using `common`?

Reply:
I do have the same opinion, I have created a package `interfaces` and moved all common interfaces there.